adhocteam/Head-Start-TTADP
# Production domain terraform

**Description of change**

* Configure production URL and external domain service (SSL cert management) via terraform

**How to test**

Visit the production URL and verify that app shows the landing page and cert is valid. You won't be able to advance past the login page.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/183

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
